src/error.rs

```diff
@@ -1,4 +1,4 @@
-use miette::Diagnostic;
+use miette::{Diagnostic, GraphicalReportHandler, GraphicalTheme};
```

Comment:
Ahh! Perfect. I knew something like `GraphicalTheme::unicode_nocolor()` should exist, but I didn't know where to start looking.